web-client/public/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wasn't able to verify this file.

Reply:
Correct, it is used for the "tiles" in windows. I can not verify it either because I'm using a Mac. But I expected it to work  because I currently don't see any issues on the GitHub related to "tiles" of the tool I used